backend/services/mgmt/v1alpha1/anonymization-service/user-account.go

```diff
@@ -0,0 +1,31 @@
+package v1alpha_anonymizationservice
+
+import (
+	"context"
+
+	"connectrpc.com/connect"
+	"github.com/jackc/pgx/v5/pgtype"
+	mgmtv1alpha1 "github.com/nucleuscloud/neosync/backend/gen/go/protos/mgmt/v1alpha1"
+	nucleuserrors "github.com/nucleuscloud/neosync/backend/internal/errors"
+	"github.com/nucleuscloud/neosync/backend/internal/neosyncdb"
+)
+
+func (s *Service) verifyUserInAccount(
+	ctx context.Context,
+	accountId string,
+) (*pgtype.UUID, error) {
+	accountUuid, err := neosyncdb.ToUuid(accountId)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
prob wanna add a check here to see if the user is an API key.
Check the similar function in the job-service.